mail_message.go

```diff
@@ -351,6 +357,24 @@ func (message *MailMessage) postToBlog(senderUser *User) error {
 	return nil
 }
 
+// Sends email data to a Mattermost webhook to post on the town-square channel
+func (message *MailMessage) postToMattermost() error  {
+	payload := fmt.Sprintf("payload = {\"channel\": \"town-square\", \"username\": \"Team 254 Mailing List Bot\", \"icon_url\": \"http://media.team254.com/2016/10/8fdb07a0-254-Swoosh.png\", \"text\": \"<!channel>\" %s}", message.body.HTML)
+	body := strings.NewReader(payload)
+	req, err := http.NewRequest("POST", "https://chat.team254.com/hooks/jxwcc5t5obbqdjcska6z39bn9w", body)
+	if err != nil {
+		return err
+	}
+	req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
```

Comment:
Is this correct? Seems like the payload is JSON. Does it still work if you set it to `application/json` or remove it?